README.md

```diff
@@ -13,6 +13,9 @@ The Puppet master is configured under Apache and Passenger by default, unless
 3.x based installation, `server_implementation` can be set to `puppetserver`
 to switch to the JVM-based Puppet Server.
 
+When using Puppet Server 2 (version 2.0 was the first version to support Puppet 4),
+Puppet Server 2.2 is the minimum version supported by this module.
```

Comment:
Nope.  For puppetserver >= 2.2 we set it to ```false``` to use the 'conf.d/auth.conf' file (which was already being installed, but never used).

For puppetserver 2.0 and 2.1, if you set ```server_use_legacy_auth_conf``` to anything, I believe you get an error when starting the service.  Error will be of the form...

```Exception in thread "main" clojure.lang.ExceptionInfo: Output of initialize-config does not match schema```
At least that's what happens in 2.3 if you try to set an unrecognised option in the same section of puppetserver.conf